src/WebUI/WorkLoads/DeploymentsTable.tsx

```diff
@@ -301,7 +301,12 @@ export class DeploymentsTable extends BaseComponent<IDeploymentsTableProperties,
         );
     }
 
-    private _openDeploymentItem = (event: React.SyntheticEvent<HTMLElement>, tableRow: ITableRow<any>, selectedItem: IDeploymentReplicaSetItem) => {
+    private _openDeploymentItem = (event, tableRow: ITableRow<any>, selectedItem: IDeploymentReplicaSetItem) => {
```

Comment:
shall we use sandbox from https://azdevui.azureedge.net/ for table and see why this code we have to write?

Reply:
please change the code for image click from 

onClick={() => this._onImageClick(imageId)}

to
onClick={(event) => {event.preventDefault(); this._onImageClick(imageId)}}

Reply:
Added e.preventDefault for all 4 loacations where we show image links